sockeye/inference.py

```diff
@@ -1249,8 +1320,8 @@ def _decode_step(self,
                      step: int,
                      source_length: int,
                      states: List[ModelState],
-                     models_output_layer_w: List[mx.nd.NDArray],
-                     models_output_layer_b: List[mx.nd.NDArray]) \
+                     models_output_layer_w: List[mx.nd.NDArray] = list(),
```

Comment:
mhh if anything we should set these to an empty tuple, as there is only one object per argument+function, so that modifying will lead to nasty side-effects.